packages/reactotron-app/App/Stores/SessionStore.js

```diff
@@ -10,6 +10,14 @@ const isSubscriptionCommandWithEmptyChanges = command => isSubscription(command)
 
 class Session {
 
+  // holds the list of commands to include in the list
+  @observable timelineCommandFilters = []
+
+  // checks if a command is on the filter list
+  isCommandFilteredOut = (command) => {
+    return this.timelineCommandFilters.filter(cmd => cmd === command.type).length === 0
```

Comment:
I couldn't quickly find a better way to run a ".contains" on a MobX observable array. Any suggestions would be fantastic!